src/SFA.DAS.EmployerIncentives/Interfaces/IRestApiClient.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Interfaces
+{
+    public interface IRestApiClient
+    {
+        Task<string> GetAsync(Uri uri, object queryData = null, CancellationToken cancellationToken = default);
```

Comment:
I think the convention here is to only use the Async suffix when there's a non-async version of the method as well?

Reply:
fixing